dev/com.ibm.jbatch.container/src/com/ibm/jbatch/container/ws/WSBatchAuthService.java

```diff
@@ -10,168 +10,175 @@
  *******************************************************************************/
 package com.ibm.jbatch.container.ws;
 
+import java.util.List;
+
 import javax.batch.operations.JobSecurityException;
 import javax.batch.operations.NoSuchJobExecutionException;
 import javax.batch.operations.NoSuchJobInstanceException;
 
 /**
- *  This is really a placeholder for the authorization service. 
- *  
- *  As we work to add the roles of submitter, monitor, and admin, we 
- *  need to flesh this out further.   
- *  
- *  Anything I write could imply to someone I've given this more thought than
- *  I have.
- *  
- *  Really the only thing that is accomplished with this is laying the
- *  groundwork for the notion that authorization will function differently
- *  in WAS than in the RI. 
- *  
- *  In WAS the JobOperator will need to use this service in addition to the WSJobManager.
+ * This is really a placeholder for the authorization service.
+ *
+ * As we work to add the roles of submitter, monitor, and admin, we
+ * need to flesh this out further.
+ *
+ * Anything I write could imply to someone I've given this more thought than
+ * I have.
+ *
+ * Really the only thing that is accomplished with this is laying the
+ * groundwork for the notion that authorization will function differently
+ * in WAS than in the RI.
+ *
+ * In WAS the JobOperator will need to use this service in addition to the WSJobManager.
  */
 public interface WSBatchAuthService {
```

Comment:
I think we've progressed beyond such a negative outlook :)   Maybe replace with an overview of the permissions needed to read vs. stop/restart/purge, or at least take out that comment.